e2e/pages/adf/contentServicesPage.ts

```diff
@@ -283,6 +323,13 @@ export class ContentServicesPage {
         return this;
     }
 
+    startUploadFile(fileLocation) {
```

Comment:
Isn't this method the same with uploadFile() method?

Reply:
Oh yes, I was just trying another approach which needed "checkUploadButton()" to be removed from the upload method. I forgot and kept it.